src/amo/components/AppBanner/styles.scss

```diff
@@ -2,10 +2,6 @@
 
 .AppBanner {
   padding: $padding-page;
-  // Split the top padding in half so that we can put optional content
```

Comment:
This was here to support both cases of showing a notice and not showing any notices at all. With your patch, there is an unwanted gap at the top when *not* showing a site notice:

<img width="362" alt="Screenshot 2019-11-12 14 40 05" src="https://user-images.githubusercontent.com/55398/68709088-94363f00-055a-11ea-96e1-8ca92ea3c5f5.png">


I was using this patch to test it:

```diff
diff --git a/src/amo/components/AppBanner/index.js b/src/amo/components/AppBanner/index.js
index 67af8682e..c4f189cdc 100644
--- a/src/amo/components/AppBanner/index.js
+++ b/src/amo/components/AppBanner/index.js
@@ -19,7 +19,6 @@ type InternalProps = {|
 export const AppBannerBase = ({ className, location }: InternalProps) => {
   return (
     <div className={makeClassName('AppBanner', className)}>
-      <SiteNotices />
       <SurveyNotice location={location} />
     </div>
   );
```